generator-prisma-client/aws-lambda-sst-esbuild/README.md

```
@@ -0,0 +1,153 @@
+
+Run:
+
+```
+pnpm prisma generate
+```
+
+### 5. Bundle the project
+
+Run
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)